src/network/networkd-link.c

```diff
@@ -2276,6 +2276,9 @@ static int link_enter_join_netdev(Link *link) {
 
         HASHMAP_FOREACH(netdev, link->network->stacked_netdevs, i) {
 
+                if (netdev->ifindex > 0)
+                        return link_joined(link);
```

Comment:
I don't know this part of the code at all, but this will terminate the loop if any of the interfaces in `stacked_netdevs` exists. This looks suspicious.

Reply:
yes right updated